src/registrar/templates/includes/summary_item.html

```diff
@@ -32,16 +28,23 @@ <h4 class="register-form-review-header">{{ sub_header_text }}</h4>
             {% include "includes/contact.html" with contact=value|first %}
           {% else %}
             {% if value %}
-              <dl class="usa-list usa-list--unstyled margin-top-0">
+              <ul class="usa-list usa-list--unstyled margin-top-0">
```

Comment:
**Question:** Could you elaborate on why we're using ul instead of dl in this case? :O

Reply:
I was messing around with the styling, but I can revert!

Reply:
Reverted!